docs/en/advanced/Scalability-Tuning.md

```diff
@@ -29,25 +29,8 @@ ALLUXIO_SECONDARY_MASTER_JAVA_OPTS+=" -Xms128g -Xmx256g "
 * As a rule of thumb set the min heap size to half the max heap size.
 * Each thread spawned by the master JVM requires off heap space determined by the thread stack size.
 When setting the heap size, ensure that there is enough memory allocated for off heap storage.
-For example, spawning `50000` threads with a default thread stack size of `1MB` requires at least
-`50 GB` of off-heap space available.
-
-### Thread Pool Size
-
-An executor pool is used on the master to handle concurrent client requests. If you expect a large
-number of concurrent clients communicating with the master, tune the thread pool size by modifying
-the following properties. The actual thread count depends on the maximum concurrency expected on the
-cluster. For example, if spawning `10000` concurrent tasks (clients) each with a client thread
-pool size of `4`, set the master thread pool max to greater than `40000`.
-
-```properties
-alluxio.master.worker.threads.max=51200
-alluxio.master.worker.threads.min=25600
-```
-
-* You may need to set OS limits, as defined in the following section, to allow the above number of
-threads to be spawned.
-* Check that the amount of off heap storage available allows for the count thread.
+For example, spawning `4000` threads with a default thread stack size of `1MB` requires at least
+`4 GB` of off-heap space available.
```

Comment:
In the line above you say "`1MB`" but the next line has "`4 GB`". I don't have a preference but I would stay consistent with spacing between number and units

Reply:
Fixed.

Reply:
Updated.